L1Trigger/L1TGlobal/plugins/TriggerMenuParser.cc

```diff
@@ -1234,7 +1234,13 @@ bool l1t::TriggerMenuParser::parseMuon(L1TUtmCondition condMu, unsigned int chip
 
         case esCutType::Eta: {
           if (etaWindows.size() < 5) {
-            etaWindows.push_back({cut.getMinimum().index, cut.getMaximum().index});
+            if ((cut.getMinimum().index <= cut.getMaximum().index) ^
+                ((cut.getMinimum().index <= 255) ^ (cut.getMaximum().index <= 255))) {
+              etaWindows.push_back({cut.getMinimum().index, cut.getMaximum().index});
+            } else {
+              edm::LogError("TriggerMenuParser")
+                  << "Invalid Eta Window for muon-condition (" << name << ")" << std::endl;
+            }
```

Comment:
Are there any consequences to issuing this error, but not having the eta window as part of the parsed condition here? Are we going to construct a seed/bit that has all the other parsed requirements, except the failed eta requirement for example?

Reply:
This does still construct the seed with all the other parsed requirements in it. Simply removing the failed requirement brought the emulator results into alignment with the firmware results for the one example of a menu with an incorrect eta window that I had. However if this is not preferred I could instead have the eta requirement placed in the condition and instead modify the eta checking code so that when an object is checked against a bad eta range it always fails as I believe this should also align the emulator results with those of the firmware.